[qtquick3d] Build failure

Package: qtquick3d[assimp,core]:arm64-osx -> 6.4.3

**Host Environment**

- Host: arm64-osx
- Compiler: AppleClang 14.0.3.14030022
-    vcpkg-tool version: 2999-12-31-unknownhash-debug
    vcpkg-scripts version: 203383666 2023-04-11 (30 hours ago)

**To Reproduce**

`vcpkg install --enforce-port-checks --allow-unsupported`

**Failure logs**

```
-- Using cached qtquick3d-everywhere-src-6.4.3.tar.xz.
-- Extracting source /Users/leanderSchulten/git_projekte/vcpkg/downloads/qtquick3d-everywhere-src-6.4.3.tar.xz
-- Using source at /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtquick3d/src/here-src-6-ee4483a0e2.clean
-- Found external ninja('1.11.1').
-- Configuring arm64-osx
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:112 (message):
    Command failed: /opt/homebrew/bin/ninja -v
    Working Directory: /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtquick3d/arm64-osx-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtquick3d/config-arm64-osx-dbg-CMakeCache.txt.log
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtquick3d/config-arm64-osx-rel-CMakeCache.txt.log
      /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtquick3d/config-arm64-osx-out.log

Call Stack (most recent call first):
  vcpkg_installed/arm64-osx/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:248 (vcpkg_execute_required_process)
  vcpkg_installed/arm64-osx/share/qtbase/qt_install_submodule.cmake:147 (vcpkg_cmake_configure)
  vcpkg_installed/arm64-osx/share/qtbase/qt_install_submodule.cmake:330 (qt_cmake_configure)
  ports/qtquick3d/portfile.cmake:21 (qt_install_submodule)
  scripts/ports.cmake:147 (include)



```
<details><summary>/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtquick3d/config-arm64-osx-out.log</summary>

```
[1/2] "/Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake" -E chdir ".." "/Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake" "/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtquick3d/src/here-src-6-ee4483a0e2.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=/Users/leanderSchulten/git_projekte/vcpkg/packages/qtquick3d_arm64-osx" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DQT_USE_DEFAULT_CMAKE_OPTIMIZATION_FLAGS:BOOL=ON" "-DQT_BUILD_EXAMPLES:BOOL=OFF" "-DQT_BUILD_TESTS:BOOL=OFF" "-DQT_BUILD_BENCHMARKS:BOOL=OFF" "-DHOST_PERL:PATH=/opt/homebrew/bin/perl" "-DINSTALL_BINDIR:STRING=bin" "-DINSTALL_LIBEXECDIR:STRING=bin" "-DINSTALL_PLUGINSDIR:STRING=Qt6/plugins" "-DINSTALL_QMLDIR:STRING=Qt6/qml" "-DFEATURE_quick3d_assimp=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_WrapQuick3DAssimp=OFF" "-DINPUT_quick3d_assimp=system" "-DTEST_quick3d_assimp=ON" "-DHAVE_Assimp=ON" "-DQT_SYNCQT:PATH=/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/tools/Qt6/bin/syncqt.pl" "-DQT_NO_FORCE_SET_CMAKE_BUILD_TYPE:BOOL=ON" "-DQT_MKSPECS_DIR:PATH=/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/share/Qt6/mkspecs" "-DCMAKE_MAKE_PROGRAM=/opt/homebrew/bin/ninja" "-DCMAKE_SYSTEM_NAME=Darwin" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=/Users/leanderSchulten/git_projekte/vcpkg/scripts/toolchains/osx.cmake" "-DVCPKG_TARGET_TRIPLET=arm64-osx" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=external" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=/Users/leanderSchulten/git_projekte/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=arm64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=/Users/leanderSchulten/git_projekte/vcpkg" "-D_VCPKG_INSTALLED_DIR=/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DCMAKE_OSX_ARCHITECTURES=arm64" "-DINSTALL_DOCDIR:STRING=doc/Qt6/" "-DINSTALL_INCLUDEDIR:STRING=include/Qt6/" "-DINSTALL_DESCRIPTIONSDIR:STRING=share/Qt6/modules" "-DINSTALL_MKSPECSDIR:STRING=share/Qt6/mkspecs" "-DINSTALL_TRANSLATIONSDIR:STRING=translations/Qt6/"
FAILED: ../CMakeCache.txt 
"/Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake" -E chdir ".." "/Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake" "/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtquick3d/src/here-src-6-ee4483a0e2.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=/Users/leanderSchulten/git_projekte/vcpkg/packages/qtquick3d_arm64-osx" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DQT_USE_DEFAULT_CMAKE_OPTIMIZATION_FLAGS:BOOL=ON" "-DQT_BUILD_EXAMPLES:BOOL=OFF" "-DQT_BUILD_TESTS:BOOL=OFF" "-DQT_BUILD_BENCHMARKS:BOOL=OFF" "-DHOST_PERL:PATH=/opt/homebrew/bin/perl" "-DINSTALL_BINDIR:STRING=bin" "-DINSTALL_LIBEXECDIR:STRING=bin" "-DINSTALL_PLUGINSDIR:STRING=Qt6/plugins" "-DINSTALL_QMLDIR:STRING=Qt6/qml" "-DFEATURE_quick3d_assimp=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_WrapQuick3DAssimp=OFF" "-DINPUT_quick3d_assimp=system" "-DTEST_quick3d_assimp=ON" "-DHAVE_Assimp=ON" "-DQT_SYNCQT:PATH=/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/tools/Qt6/bin/syncqt.pl" "-DQT_NO_FORCE_SET_CMAKE_BUILD_TYPE:BOOL=ON" "-DQT_MKSPECS_DIR:PATH=/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/share/Qt6/mkspecs" "-DCMAKE_MAKE_PROGRAM=/opt/homebrew/bin/ninja" "-DCMAKE_SYSTEM_NAME=Darwin" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=/Users/leanderSchulten/git_projekte/vcpkg/scripts/toolchains/osx.cmake" "-DVCPKG_TARGET_TRIPLET=arm64-osx" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=external" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=/Users/leanderSchulten/git_projekte/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=arm64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=/Users/leanderSchulten/git_projekte/vcpkg" "-D_VCPKG_INSTALLED_DIR=/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DCMAKE_OSX_ARCHITECTURES=arm64" "-DINSTALL_DOCDIR:STRING=doc/Qt6/" "-DINSTALL_INCLUDEDIR:STRING=include/Qt6/" "-DINSTALL_DESCRIPTIONSDIR:STRING=share/Qt6/modules" "-DINSTALL_MKSPECSDIR:STRING=share/Qt6/mkspecs" "-DINSTALL_TRANSLATIONSDIR:STRING=translations/Qt6/"
-- The CXX compiler identification is AppleClang 14.0.3.14030022
-- The C compiler identification is AppleClang 14.0.3.14030022
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE  
-- Found ZLIB: optimized;/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libz.a;debug;/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libz.a (found version "1.2.13") 
-- Found WrapSystemZLIB: TRUE  
-- Using system ZLIB.
-- Found WrapZLIB: TRUE  
-- Performing Test HAVE_STDATOMIC
-- Performing Test HAVE_STDATOMIC - Success
-- Found WrapAtomic: TRUE  
-- Looking for backtrace
-- Looking for backtrace - found
-- backtrace facility detected in default set of libraries
-- Found Backtrace: /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include  
-- Found WrapSystemDoubleConversion: /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/share/double-conversion/double-conversionConfig.cmake (found version "3.2.0") 
-- Found Libb2: /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libb2.a  
-- Found the following ICU libraries:
--   i18n (required): /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libicui18n.a
--   uc (required): /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libicuuc.a
--   data (required): /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libicudata.a
-- Found ICU: /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include (found version "72.1") 
-- Found the following ICU libraries:
--   i18n (required): /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libicui18n.a
--   uc (required): /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libicuuc.a
--   data (required): /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libicudata.a
-- Found ICU: /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include (found version "72.1") 
-- Found the following ICU libraries:
--   i18n (required): /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libicui18n.a
--   uc (required): /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libicuuc.a
--   data (required): /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libicudata.a
-- Found ICU: /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include (found version "72.1") 
-- Found WrapSystemPCRE2: /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libpcre2-16.a  
-- Using system PCRE2.
-- Found WrapPCRE2: TRUE  
-- Found WrapZSTD: 1.5.4 (found suitable version "1.5.4", minimum required is "1.3") 
-- Performing Test HAVE_GETTIME
-- Performing Test HAVE_GETTIME - Success
-- Found WrapRt: TRUE  
-- Found OpenSSL: /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libcrypto.a (found version "3.1.0")  
-- Found WrapOpenSSLHeaders: /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include (found version "3.1.0") 
-- Found OpenGL: /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/System/Library/Frameworks/OpenGL.framework   
-- Found WrapSystemPNG: optimized;/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libpng16.a;debug;/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libpng16d.a;optimized;/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libz.a;debug;/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libz.a (found version "1.6.39") 
-- Using system PNG.
-- Found BZip2: optimized;/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libbz2.a;debug;/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libbz2d.a (found version "1.0.8") 
-- Looking for BZ2_bzCompressInit
-- Looking for BZ2_bzCompressInit - found
-- Found PNG: optimized;/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libpng16.a;debug;/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libpng16d.a (found version "1.6.39") 
-- Found WrapSystemHarfbuzz: TRUE  
-- Using system Harfbuzz.
-- Found WrapSystemFreetype: TRUE (found version "2.12.1") 
-- Using system Freetype.
-- Found WrapSystemJpeg: optimized;/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libjpeg.a;debug;/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libjpeg.a (found version "62") 
-- Using system Jpeg.
CMake Error at /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/share/Qt6/QtPublicTargetHelpers.cmake:257 (set_property):
  Attempt to promote imported target "ZLIB::ZLIB" to global scope (by setting
  IMPORTED_GLOBAL) which is not built in this directory.
Call Stack (most recent call first):
  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/share/Qt6/QtPublicWalkLibsHelpers.cmake:252 (__qt_internal_promote_target_to_global)
  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/share/Qt6/QtPublicWalkLibsHelpers.cmake:225 (__qt_internal_walk_libs)
  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/share/Qt6/QtPublicWalkLibsHelpers.cmake:225 (__qt_internal_walk_libs)
  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/share/Qt6/QtFindPackageHelpers.cmake:12 (__qt_internal_walk_libs)
  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/share/Qt6/QtFindPackageHelpers.cmake:181 (qt_find_package_promote_targets_to_global_scope)
  src/plugins/assetimporters/configure.cmake:13 (qt_find_package)
  src/plugins/assetimporters/CMakeLists.txt:9 (include)



-- Configuration summary shown below. It has also been written to /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtquick3d/arm64-osx-rel/config.summary
-- Configure with --log-level=STATUS or higher to increase CMake's message verbosity. The log level does not persist across reconfigurations.
 
-- Configure summary:

QtQuick3D:
  Assimp ................................. yes
  System Assimp .......................... yes
 

-- Configuring incomplete, errors occurred!
See also "/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtquick3d/arm64-osx-rel/CMakeFiles/CMakeOutput.log".
See also "/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtquick3d/arm64-osx-rel/CMakeFiles/CMakeError.log".
[2/2] "/Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake" -E chdir "../../arm64-osx-dbg" "/Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake" "/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtquick3d/src/here-src-6-ee4483a0e2.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=/Users/leanderSchulten/git_projekte/vcpkg/packages/qtquick3d_arm64-osx/debug" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DQT_USE_DEFAULT_CMAKE_OPTIMIZATION_FLAGS:BOOL=ON" "-DQT_BUILD_EXAMPLES:BOOL=OFF" "-DQT_BUILD_TESTS:BOOL=OFF" "-DQT_BUILD_BENCHMARKS:BOOL=OFF" "-DHOST_PERL:PATH=/opt/homebrew/bin/perl" "-DINSTALL_BINDIR:STRING=bin" "-DINSTALL_LIBEXECDIR:STRING=bin" "-DINSTALL_PLUGINSDIR:STRING=Qt6/plugins" "-DINSTALL_QMLDIR:STRING=Qt6/qml" "-DFEATURE_quick3d_assimp=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_WrapQuick3DAssimp=OFF" "-DINPUT_quick3d_assimp=system" "-DTEST_quick3d_assimp=ON" "-DHAVE_Assimp=ON" "-DQT_SYNCQT:PATH=/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/tools/Qt6/bin/syncqt.pl" "-DQT_NO_FORCE_SET_CMAKE_BUILD_TYPE:BOOL=ON" "-DQT_MKSPECS_DIR:PATH=/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/share/Qt6/mkspecs" "-DCMAKE_MAKE_PROGRAM=/opt/homebrew/bin/ninja" "-DCMAKE_SYSTEM_NAME=Darwin" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=/Users/leanderSchulten/git_projekte/vcpkg/scripts/toolchains/osx.cmake" "-DVCPKG_TARGET_TRIPLET=arm64-osx" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=external" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=/Users/leanderSchulten/git_projekte/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=arm64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=/Users/leanderSchulten/git_projekte/vcpkg" "-D_VCPKG_INSTALLED_DIR=/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DCMAKE_OSX_ARCHITECTURES=arm64" "-DINSTALL_DOCDIR:STRING=../doc/Qt6/" "-DINSTALL_INCLUDEDIR:STRING=../include/Qt6/" "-DINSTALL_TRANSLATIONSDIR:STRING=../translations/Qt6/" "-DINSTALL_DESCRIPTIONSDIR:STRING=../share/Qt6/modules" "-DINSTALL_MKSPECSDIR:STRING=../share/Qt6/mkspecs"
FAILED: ../../arm64-osx-dbg/CMakeCache.txt 
"/Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake" -E chdir "../../arm64-osx-dbg" "/Users/leanderSchulten/git_projekte/vcpkg/downloads/tools/cmake-3.25.1-osx/cmake-3.25.1-macos-universal/CMake.app/Contents/bin/cmake" "/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtquick3d/src/here-src-6-ee4483a0e2.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=/Users/leanderSchulten/git_projekte/vcpkg/packages/qtquick3d_arm64-osx/debug" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON" "-DQT_USE_DEFAULT_CMAKE_OPTIMIZATION_FLAGS:BOOL=ON" "-DQT_BUILD_EXAMPLES:BOOL=OFF" "-DQT_BUILD_TESTS:BOOL=OFF" "-DQT_BUILD_BENCHMARKS:BOOL=OFF" "-DHOST_PERL:PATH=/opt/homebrew/bin/perl" "-DINSTALL_BINDIR:STRING=bin" "-DINSTALL_LIBEXECDIR:STRING=bin" "-DINSTALL_PLUGINSDIR:STRING=Qt6/plugins" "-DINSTALL_QMLDIR:STRING=Qt6/qml" "-DFEATURE_quick3d_assimp=ON" "-DCMAKE_DISABLE_FIND_PACKAGE_WrapQuick3DAssimp=OFF" "-DINPUT_quick3d_assimp=system" "-DTEST_quick3d_assimp=ON" "-DHAVE_Assimp=ON" "-DQT_SYNCQT:PATH=/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/tools/Qt6/bin/syncqt.pl" "-DQT_NO_FORCE_SET_CMAKE_BUILD_TYPE:BOOL=ON" "-DQT_MKSPECS_DIR:PATH=/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/share/Qt6/mkspecs" "-DCMAKE_MAKE_PROGRAM=/opt/homebrew/bin/ninja" "-DCMAKE_SYSTEM_NAME=Darwin" "-DBUILD_SHARED_LIBS=OFF" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=/Users/leanderSchulten/git_projekte/vcpkg/scripts/toolchains/osx.cmake" "-DVCPKG_TARGET_TRIPLET=arm64-osx" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=external" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=/Users/leanderSchulten/git_projekte/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=arm64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=/Users/leanderSchulten/git_projekte/vcpkg" "-D_VCPKG_INSTALLED_DIR=/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DCMAKE_OSX_ARCHITECTURES=arm64" "-DINSTALL_DOCDIR:STRING=../doc/Qt6/" "-DINSTALL_INCLUDEDIR:STRING=../include/Qt6/" "-DINSTALL_TRANSLATIONSDIR:STRING=../translations/Qt6/" "-DINSTALL_DESCRIPTIONSDIR:STRING=../share/Qt6/modules" "-DINSTALL_MKSPECSDIR:STRING=../share/Qt6/mkspecs"
-- The CXX compiler identification is AppleClang 14.0.3.14030022
-- The C compiler identification is AppleClang 14.0.3.14030022
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE  
-- Found ZLIB: optimized;/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libz.a;debug;/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libz.a (found version "1.2.13") 
-- Found WrapSystemZLIB: TRUE  
-- Using system ZLIB.
-- Found WrapZLIB: TRUE  
-- Performing Test HAVE_STDATOMIC
-- Performing Test HAVE_STDATOMIC - Success
-- Found WrapAtomic: TRUE  
-- Looking for backtrace
-- Looking for backtrace - found
-- backtrace facility detected in default set of libraries
-- Found Backtrace: /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/usr/include  
-- Found WrapSystemDoubleConversion: /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/share/double-conversion/double-conversionConfig.cmake (found version "3.2.0") 
-- Found Libb2: /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libb2.a  
-- Found the following ICU libraries:
--   i18n (required): /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libicui18n.a
--   uc (required): /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libicuuc.a
--   data (required): /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libicudata.a
-- Found ICU: /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include (found version "72.1") 
-- Found the following ICU libraries:
--   i18n (required): /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libicui18n.a
--   uc (required): /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libicuuc.a
--   data (required): /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libicudata.a
-- Found ICU: /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include (found version "72.1") 
-- Found the following ICU libraries:
--   i18n (required): /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libicui18n.a
--   uc (required): /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libicuuc.a
--   data (required): /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libicudata.a
-- Found ICU: /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include (found version "72.1") 
-- Found WrapSystemPCRE2: /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libpcre2-16.a  
-- Using system PCRE2.
-- Found WrapPCRE2: TRUE  
-- Found WrapZSTD: 1.5.4 (found suitable version "1.5.4", minimum required is "1.3") 
-- Performing Test HAVE_GETTIME
-- Performing Test HAVE_GETTIME - Success
-- Found WrapRt: TRUE  
-- Found OpenSSL: /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libcrypto.a (found version "3.1.0")  
-- Found WrapOpenSSLHeaders: /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/include (found version "3.1.0") 
-- Found OpenGL: /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX13.3.sdk/System/Library/Frameworks/OpenGL.framework   
-- Found WrapSystemPNG: optimized;/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libpng16.a;debug;/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libpng16d.a;optimized;/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libz.a;debug;/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libz.a (found version "1.6.39") 
-- Using system PNG.
-- Found BZip2: optimized;/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libbz2.a;debug;/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libbz2d.a (found version "1.0.8") 
-- Looking for BZ2_bzCompressInit
-- Looking for BZ2_bzCompressInit - found
-- Found PNG: optimized;/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libpng16.a;debug;/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libpng16d.a (found version "1.6.39") 
-- Found WrapSystemHarfbuzz: TRUE  
-- Using system Harfbuzz.
-- Found WrapSystemFreetype: TRUE (found version "2.12.1") 
-- Using system Freetype.
-- Found WrapSystemJpeg: optimized;/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libjpeg.a;debug;/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libjpeg.a (found version "62") 
-- Using system Jpeg.
CMake Error at /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/share/Qt6/QtPublicTargetHelpers.cmake:257 (set_property):
  Attempt to promote imported target "ZLIB::ZLIB" to global scope (by setting
  IMPORTED_GLOBAL) which is not built in this directory.
Call Stack (most recent call first):
  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/share/Qt6/QtPublicWalkLibsHelpers.cmake:252 (__qt_internal_promote_target_to_global)
  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/share/Qt6/QtPublicWalkLibsHelpers.cmake:225 (__qt_internal_walk_libs)
  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/share/Qt6/QtPublicWalkLibsHelpers.cmake:225 (__qt_internal_walk_libs)
  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/share/Qt6/QtFindPackageHelpers.cmake:12 (__qt_internal_walk_libs)
  /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/share/Qt6/QtFindPackageHelpers.cmake:181 (qt_find_package_promote_targets_to_global_scope)
  src/plugins/assetimporters/configure.cmake:13 (qt_find_package)
  src/plugins/assetimporters/CMakeLists.txt:9 (include)



-- Configuration summary shown below. It has also been written to /Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtquick3d/arm64-osx-dbg/config.summary
-- Configure with --log-level=STATUS or higher to increase CMake's message verbosity. The log level does not persist across reconfigurations.
 
-- Configure summary:

QtQuick3D:
  Assimp ................................. yes
  System Assimp .......................... yes
 

-- Configuring incomplete, errors occurred!
See also "/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtquick3d/arm64-osx-dbg/CMakeFiles/CMakeOutput.log".
See also "/Users/leanderSchulten/git_projekte/vcpkg/buildtrees/qtquick3d/arm64-osx-dbg/CMakeFiles/CMakeError.log".
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "$schema": "https://raw.githubusercontent.com/microsoft/vcpkg-tool/main/docs/vcpkg.schema.json",
  "name": "lichtsteuerung",
  "version-string": "2.0 $.0",
  "dependencies": [
    {
      "name": "qtquick3d",
      "default-features": false,
      "features": [
        "assimp"
      ]
    }
  ]
}

```
</details>

```
CMake Error at /Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/share/Qt6/QtPublicTargetHelpers.cmake:257 (set_property):
  Attempt to promote imported target "ZLIB::ZLIB" to global scope (by setting
  IMPORTED_GLOBAL) which is not built in this directory.
```
Is it a system or internal conflict with the `zlib` provided by vcpkg? 🤔

```
-- Found ZLIB: optimized;/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/lib/libz.a;debug;/Users/leanderSchulten/git_projekte/vcpkg/vcpkg_installed/arm64-osx/debug/lib/libz.a (found version "1.2.13") 
```
QT:
```
//Qt feature: system_zlib (from target Qt6::Core)
QT_FEATURE_system_zlib:INTERNAL=ON
```
VCPKG:
```
VCPKG_MANIFEST_INSTALL:INTERNAL=OFF
//ADVANCED property for variable: VCPKG_VERBOSE
VCPKG_VERBOSE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: ZLIB_INCLUDE_DIR
ZLIB_INCLUDE_DIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: ZLIB_LIBRARY_DEBUG
ZLIB_LIBRARY_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: ZLIB_LIBRARY_RELEASE
ZLIB_LIBRARY_RELEASE-ADVANCED:INTERNAL=1
```

--- a/ports/qtquick3d/portfile.cmake
+++ b/ports/qtquick3d/portfile.cmake
@@ -22,6 +22,7 @@ qt_install_submodule(PATCHES    ${${PORT}_PATCHES}
                      TOOL_NAMES ${TOOL_NAMES}
                      CONFIGURE_OPTIONS
                         ${FEATURE_OPTIONS}
+                        -DCMAKE_FIND_PACKAGE_TARGETS_GLOBAL=ON
                      CONFIGURE_OPTIONS_RELEASE
                      CONFIGURE_OPTIONS_DEBUG
                     )

--- a/ports/qtquick3d/vcpkg.json
+++ b/ports/qtquick3d/vcpkg.json
@@ -1,6 +1,7 @@
 {
   "name": "qtquick3d",
   "version": "6.4.3",
+  "port-version": 1,
   "description": "A new module and API for defining 3D content in Qt Quick.",
   "homepage": "https://www.qt.io/",
   "license": null,

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -6694,7 +6694,7 @@
     },
     "qtquick3d": {
       "baseline": "6.4.3",
-      "port-version": 0
+      "port-version": 1
     },
     "qtquick3dphysics": {
       "baseline": "6.4.3",

--- a/versions/q-/qtquick3d.json
+++ b/versions/q-/qtquick3d.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "a927a30de733a8bb6f1149dd4a5a37aeeca8f521",
+      "version": "6.4.3",
+      "port-version": 1
+    },
+    {
       "git-tree": "4fcbc24a4f09f2e43f69d771ed87ac6cc187a629",
       "version": "6.4.3",
       "port-version": 0

